spec/abci/abci++_basic_concepts.md

```diff
@@ -16,12 +16,11 @@ title: Overview and basic concepts
   - [Proposal timeout](#proposal-timeout)
   - [Deterministic State-Machine Replication](#deterministic-state-machine-replication)
   - [Events](#events)
-  - [Evidence](#evidence)
+  - [Evidences of Misbehavior](#evidences-of-misbehavior)
```

Comment:
I think that Evidences is not a word, see: 

https://github.com/cometbft/cometbft/blob/3301699a68f477bfea7e7c048cf955b88e6062b6/types/evidence.go#L447-L448

Reply:
thanks, fixed in all other places too